helper/schema/schema_test.go

```diff
@@ -3947,6 +3947,33 @@ func TestSchemaMap_Validate(t *testing.T) {
 			Err: false,
 		},
 
+		"Good sub-resource, interpolated value": {
```

Comment:
It's a bit unclear to me what `IsComputed` is actually checking for. Can you provide me with an example? Is this something like, `foo.0`?